src/cargo/ops/resolve.rs

```diff
@@ -387,6 +387,8 @@ pub fn resolve_with_previous<'gctx>(
         register_previous_locks(ws, registry, r, &keep, dev_deps);
 
         // Prefer to use anything in the previous lock file, aka we want to have conservative updates.
+        let span = tracing::span!(tracing::Level::TRACE, "prefer_package_id");
```

Comment:
You can use [`entered()`](https://docs.rs/tracing/latest/tracing/span/struct.Span.html#method.entered) here and avoid the extra `_enter` variable below.

```suggestion
        let _span = tracing::span!(tracing::Level::TRACE, "prefer_package_id").entered();
```

Reply:
I like how none of the examples reference that